physics/MP/NSSL/module_mp_nssl_2mom.F90

```diff
@@ -5929,6 +6078,104 @@ SUBROUTINE calc_eff_radius    &
          ENDIF
          ENDIF
 
+         IF ( present(t5) .and. flag_t5 ) THEN
+         
+         ! first: case when hail is off
+          
+         IF (  lhl < 1 .or. flag_t6 ) THEN
+         ! graupel only 
+           IF ( qx(mgs,lh) > Max(1.e-8,qxmin(lh)) ) THEN
+            ! Lambda for graupel
+            hwdn = xdn0(lh)
+            IF ( lvh > 1 ) THEN ! variable density
+              IF ( an(ix,jy,kz,lvh) > 1.e-30 ) THEN
```

Comment:
Suggestion: you may want to define a variable for `1.e-30` in case it ever needs to change.

Reply:
Yeah, easy to do and makes sure things are consistent. I can do it for this or the next update.